lib/queue_classic_plus/update_metrics.rb

```diff
@@ -18,7 +18,7 @@ def self.metrics
       {
         jobs_queued: jobs_queued,
         jobs_scheduled: jobs_scheduled,
-        max_created_at: max_age("created_at"),
+        max_created_at: max_age("created_at", "created_at = scheduled_at"),
```

Comment:
But yeah, this could be made more robust, by adding a couple of seconds before and after?